sdk/monitor/monitor-ingestion/README.md

```diff
@@ -242,6 +242,28 @@ module.exports = { main };
 
 Logs uploaded using the Monitor Ingestion client library can be retrieved using the [Monitor Query client library][monitor_query].
 
+#### Configure client for Azure sovereign cloud
```

Comment:
Move this to a subsection within the "Authenticate the client" section. Example here for Monitor Query: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/monitor/monitor-query/README.md#configure-client-for-azure-sovereign-cloud

Reply:
@scottaddie  Done